Support for project branches (#109)
lequal/sonar-cnes-report: README.md

#### Get help
Use `java -jar cnesreport.jar -h` to get the following help about cnesreport:
````
usage: java -jar cnesreport.jar [-a <arg>] [-b <arg>] [-c] [-d <arg>] [-e] [-h] [-l <arg>] [-o <arg>] [-p <arg>] [-r <arg>]
       [-s <arg>] [-t <arg>] [-v] [-w] [-x <arg>]
Generate editable reports for SonarQube projects.

 -a,--author <arg>                 Name of the report writer.

>  -b,--branch <arg>                 Branch of the targeted project. Requires Developer Edition or sonarqube-community-branch-plugin.

Reviewer: Could you please write what the default value is ?
Just like the "date" parameter for example.

Author: Default is nothing, which means *do not use branches*. How do you think, we should call this default value? `(defaults to using no branches)`?

Reviewer: I asked because I thought that maybe changing the default to master would be useful.
It may be useless in the end, I agree.

Author: Behaviour:
**SonarQube with branches-support**
`-b` is set --> the branch of the project (`-p`) is used
`-b` is not set --> the *main branch* of the project (`-p`) is used (mostly `master` but can be changed in SQ)
**SonarQube without branches-support**
`-b` is set --> undefined behaviour
`-b` is not set --> the project (`-p`) is used

Author: ```suggestion
 -b,--branch <arg>                 Branch of the targeted project. Requires Developer Edition or sonarqube-community-branch-plugin. Default: usage of main branch.
```

Reviewer: Thanks, looks good to me.

Reviewer: Could you please commit the suggested changes ?  
Then I will validate the pull request.

java -jar cnesreport.jar -t xuixg5hub345xbefu -s https://example.org:9000 -p projectId -r ./template.docx
````

##### Export of a specific project branch (standalone)

> If you are using a commercial edition of sonarqube or the [sonarqube-community-branch-plugin](https://github.com/mc1arke/sonarqube-community-branch-plugin) you want to export the report for a specific branch of your project using the -b option. 

Reviewer: Could you please replace " **you want to** export the report" by "**, you can** export the report" ?
Just in order to match the sentence format of the other sections.

Author: ```suggestion
If you are using a commercial edition of sonarqube or the [sonarqube-community-branch-plugin](https://github.com/mc1arke/sonarqube-community-branch-plugin) you can export the report for a specific branch of your project using the -b option. 
```
